infra/cmake/modules/ExternalSourceTools.cmake

```diff
@@ -53,17 +53,10 @@ function(ExternalSource_Download PREFIX)
 
       # For external mirror server
       envoption(EXTERNAL_SERVER_USERPWD "")
-
-      if("${EXTERNAL_SERVER_USERPWD}" STREQUAL "")
-        file(DOWNLOAD ${URL} "${DOWNLOAD_PATH}"
-                      STATUS status
-                      USERPWD "${ARG_USERPWD}"
-                      LOG log)
-      else("${EXTERNAL_SERVER_USERPWD}" STREQUAL "")
-        file(DOWNLOAD ${URL} "${DOWNLOAD_PATH}"
-                      STATUS status
-                      LOG log)
-      endif("${EXTERNAL_SERVER_USERPWD}" STREQUAL "")
+      file(DOWNLOAD ${URL} "${DOWNLOAD_PATH}"
+                    STATUS status
+                    USERPWD "${EXTERNAL_SERVER_USERPWD}"
+                    LOG log)
```

Comment:
@hseok-oh I just confirm my understanding is correct.

> This commit fixes invalid variable name in ExternalSourceTools.

I don't find invalid variable name in code change. I guessed there was a typo fix.
It seems to remove setting via `ARG_USERPWD` when env variable`EXTERNAL_SERVER_USERPWD` is "".
Maybe `ARG_USERPWD` is not used any longer.

Reply:
>Maybe `ARG_USERPWD` is not used any longer.

Right